docs/installation.md

```diff
@@ -218,19 +218,208 @@ On Linux, as an optional post-installation task, you may wish to setup the bot t
 
 ------
 
-### Anaconda
+## Conda (Miniconda or Anaconda)
 
-Freqtrade can also be installed using Anaconda (or Miniconda).
+Freqtrade can also be installed with Miniconda or Anaconda. Conda (Miniconda or Anaconda) would automatically prepare and manage the extensive library-dependencies of the Freqtrade program.
 
-!!! Note
-    This requires the [ta-lib](#1-install-ta-lib) C-library to be installed first. See below.
+##### What is Conda?
+
+It is: (1) package, (2) dependency and (3) environment management for any programming language : https://docs.conda.io/projects/conda/en/latest/index.html
+
+Shortly : Conda < Miniconda < Anaconda. Check : https://linuxnetmag.com/miniconda-vs-anaconda/
+
+It is recommended to install Miniconda, not Anaconda. The difference between both lies in the amount of packages that would be installed upfront. Difference is at least like 1:8.
+
+https://www.dunderdata.com/blog/anaconda-is-bloated-set-up-a-lean-robust-data-science-environment-with-miniconda-and-conda-forge
+
+
+#### 1. Instal Conda
+
+3 step installation instruction can be found here :
+https://conda.io/projects/conda/en/latest/user-guide/install/linux.html#install-linux-silent
+
+Confirm with `yes` all questions. After installation, it is mandatory to turn your terminal OFF and ON again.
+
+
+#### 2. Prepare Conda environment
+
+##### Change the channels with upgrades:
+
+After opening terminal, you already will be in default `base` conda environment
+
+If you want, you can prevent the (base) conda environment from being activated automatically.
+
+```bash
+conda config --set auto_activate_base false
+```
+
+enter the conda base environment:
+
+```bash
+conda activate base
+```
+
+Conda as a package manager can download new packages from "channels". The best developed Conda channel, is not the default channel and is called `conda-forge`. The code below switches to it.
+
+```bash    
+# adding forge
+conda config --env --add channels conda-forge
+
+# make it strict
+conda config --env --set channel_priority strict 	
+```
+
+You can check the status of conda with the following code.
+
+```bash
+conda info
+conda config --show channels
+conda config --show channel_priority
+```
+
+#### 3. Freqtrade Conda Environment
+
+Now you have conda, but only (base) environment,
 
-``` bash
-conda env create -f environment.yml
+```bash
+conda env list
+```
+
+It is time to setup environment of the Freqtrade itself:
+
+The conda command `create -n` automatically installs all nested dependencies for the selected libraries
+
+general structure of installation command is:
+
+```bash
+conda create -n [name of the environment] [python version] [packages]
 ```
 
+so it can be
+
+```bash
+conda create -n freqtrade-conda python=3.8 pandas numpy ta-lib git wheel virtualenv
```

Comment:
Whcih version of ta-lib will be installed when using it like this?

[searching conda ](https://anaconda.org/search?q=ta-lib) reveals quite a few packages which could match ... 
are we sure it's going to pick the right one (i assume it's [this one](https://anaconda.org/conda-forge/ta-lib/files)) - and are there any details to be found HOW this is actually built?

I'm asking this because we do a small change to the ta-lib C sourcecode ([click to see the modification](https://github.com/freqtrade/freqtrade/blob/develop/build_helpers/install_ta-lib.sh#L10) ) before installing the C library to make sure there are no problems when trading BTC Pairs (they often have very low decimal prices - and ta-lib tends to get inprecise as by default, it only works with a limited number of decimal places).

Reply:
There are several ways to select Ta-Lib.
The first is the update channel, in this case we have conda-forge set up, so yes, you were right, it is that one.
https://anaconda.org/conda-forge/ta-lib

So parameters here are OS, python version and ta-lib version. 
Conda-forge installs ta-lib v0.4.19 by default, but the repository also has v0.4.18

How they are built in detail, we would have to ask the developers on the other side of this mirror.

Reply:
Thinking about this today i think this is problematic.

while ta-lib is installed with Conda (from conda-forge) - i'm not sure it's the one that's used if i follow this tutorial, as you're using the `./setup.sh --install` script at the moment.

Part of this script installs ta-lib from source (followed by the python dependency).

you'll most likely end up with:
* the conda version in the conda environment
* the C library installed in the system (but most likely unused - see below)
* pip install: (this will need a detailed log-analysis)
  * my gut tells me that pip detects that ta-lib is available - so it's not installing it in the virtual environment. 
  this is problematic, as the decimal fix will then be installed as C library - but will not be used by ta-lib, since the conda package will (hopefully) bundle the C library as well.


Now this makes the whole installation method some very hacky and dirty solution - as it installs a system-wide dependency but does not use that, but instead uses a ta-lib dependency, which however can cause problems to your strategy depending on the pairs you're using.

Reply:
#1 install Conda and prepare Conva-venv(base) 


```
black@mx:~
$ conda config --env --add channels conda-forge 
(base) black@mx:~
$ conda config --env --set channel_priority strict 
(base) black@mx:~
$ conda create -n freqtrader-env python=3.8 pandas numpy ta-lib git wheel virtualenv
Collecting package metadata (current_repodata.json): done
Solving environment: done

## Package Plan ##

  environment location: /home/black/miniconda3/envs/freqtrader-env

  added / updated specs:
    - git
    - numpy
    - pandas
    - python=3.8
    - ta-lib
    - virtualenv
    - wheel


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    _libgcc_mutex-0.1          |      conda_forge           3 KB  conda-forge
    _openmp_mutex-4.5          |            1_gnu          22 KB  conda-forge
    appdirs-1.4.4              |     pyh9f0ad1d_0          13 KB  conda-forge
    c-ares-1.17.1              |       h36c2ea0_0         111 KB  conda-forge
    ca-certificates-2020.12.5  |       ha878542_0         137 KB  conda-forge
    certifi-2020.12.5          |   py38h578d9bd_1         143 KB  conda-forge
    curl-7.71.1                |       he644dc0_8         139 KB  conda-forge
    distlib-0.3.1              |     pyh9f0ad1d_0         246 KB  conda-forge
    expat-2.2.9                |       he1b5a44_2         191 KB  conda-forge
    filelock-3.0.12            |     pyh9f0ad1d_0          10 KB  conda-forge
    gettext-0.19.8.1           |    h0b5b191_1005         3.6 MB  conda-forge
    git-2.30.0                 | pl5320h014a29a_0        12.5 MB  conda-forge
    krb5-1.17.2                |       h926e7f8_0         1.4 MB  conda-forge
    ld_impl_linux-64-2.35.1    |       hea4e1c9_1         617 KB  conda-forge
    libblas-3.9.0              |       7_openblas          11 KB  conda-forge
    libcblas-3.9.0             |       7_openblas          11 KB  conda-forge
    libcurl-7.71.1             |       hcdd3856_8         312 KB  conda-forge
    libedit-3.1.20191231       |       he28a2e2_2         121 KB  conda-forge
    libev-4.33                 |       h516909a_1         104 KB  conda-forge
    libffi-3.3                 |       h58526e2_2          51 KB  conda-forge
    libgcc-ng-9.3.0            |      h2828fa1_18         7.8 MB  conda-forge
    libgfortran-ng-9.3.0       |      hff62375_18          22 KB  conda-forge
    libgfortran5-9.3.0         |      hff62375_18         2.0 MB  conda-forge
    libgomp-9.3.0              |      h2828fa1_18         376 KB  conda-forge
    libiconv-1.16              |       h516909a_0         1.4 MB  conda-forge
    liblapack-3.9.0            |       7_openblas          11 KB  conda-forge
    libnghttp2-1.41.0          |       h8cfc5f6_2         774 KB  conda-forge
    libopenblas-0.3.12         |pthreads_h4812303_1         8.9 MB  conda-forge
    libssh2-1.9.0              |       hab1572f_5         225 KB  conda-forge
    libstdcxx-ng-9.3.0         |      h6de172a_18         4.0 MB  conda-forge
    libta-lib-0.4.0            |       h516909a_0         503 KB  conda-forge
    ncurses-6.2                |       h58526e2_4         985 KB  conda-forge
    numpy-1.19.5               |   py38h18fd61f_1         5.4 MB  conda-forge
    openssl-1.1.1i             |       h7f98852_0         2.1 MB  conda-forge
    pandas-1.2.0               |   py38h51da96c_1        12.0 MB  conda-forge
    pcre-8.44                  |       he1b5a44_0         261 KB  conda-forge
    perl-5.32.0                |       h36c2ea0_0        16.8 MB  conda-forge
    pip-20.3.3                 |     pyhd8ed1ab_0         1.1 MB  conda-forge
    python-3.8.6               |hffdb5ce_4_cpython        26.1 MB  conda-forge
    python-dateutil-2.8.1      |             py_0         220 KB  conda-forge
    python_abi-3.8             |           1_cp38           4 KB  conda-forge
    pytz-2020.5                |     pyhd8ed1ab_0         244 KB  conda-forge
    readline-8.0               |       he28a2e2_2         281 KB  conda-forge
    setuptools-49.6.0          |   py38h578d9bd_3         929 KB  conda-forge
    six-1.15.0                 |     pyh9f0ad1d_0          14 KB  conda-forge
    sqlite-3.34.0              |       h74cdb3f_0         1.4 MB  conda-forge
    ta-lib-0.4.19              |   py38hab2c0dc_2         425 KB  conda-forge
    tk-8.6.10                  |       h21135ba_1         3.2 MB  conda-forge
    virtualenv-20.4.0          |   py38h578d9bd_0         5.0 MB  conda-forge
    wheel-0.36.2               |     pyhd3deb0d_0          31 KB  conda-forge
    xz-5.2.5                   |       h516909a_1         343 KB  conda-forge
    zlib-1.2.11                |    h516909a_1010         106 KB  conda-forge
    ------------------------------------------------------------
                                           Total:       122.6 MB

The following NEW packages will be INSTALLED:

  _libgcc_mutex      conda-forge/linux-64::_libgcc_mutex-0.1-conda_forge
  _openmp_mutex      conda-forge/linux-64::_openmp_mutex-4.5-1_gnu
  appdirs            conda-forge/noarch::appdirs-1.4.4-pyh9f0ad1d_0
  c-ares             conda-forge/linux-64::c-ares-1.17.1-h36c2ea0_0
  ca-certificates    conda-forge/linux-64::ca-certificates-2020.12.5-ha878542_0
  certifi            conda-forge/linux-64::certifi-2020.12.5-py38h578d9bd_1
  curl               conda-forge/linux-64::curl-7.71.1-he644dc0_8
  distlib            conda-forge/noarch::distlib-0.3.1-pyh9f0ad1d_0
  expat              conda-forge/linux-64::expat-2.2.9-he1b5a44_2
  filelock           conda-forge/noarch::filelock-3.0.12-pyh9f0ad1d_0
  gettext            conda-forge/linux-64::gettext-0.19.8.1-h0b5b191_1005
  git                conda-forge/linux-64::git-2.30.0-pl5320h014a29a_0
  krb5               conda-forge/linux-64::krb5-1.17.2-h926e7f8_0
  ld_impl_linux-64   conda-forge/linux-64::ld_impl_linux-64-2.35.1-hea4e1c9_1
  libblas            conda-forge/linux-64::libblas-3.9.0-7_openblas
  libcblas           conda-forge/linux-64::libcblas-3.9.0-7_openblas
  libcurl            conda-forge/linux-64::libcurl-7.71.1-hcdd3856_8
  libedit            conda-forge/linux-64::libedit-3.1.20191231-he28a2e2_2
  libev              conda-forge/linux-64::libev-4.33-h516909a_1
  libffi             conda-forge/linux-64::libffi-3.3-h58526e2_2
  libgcc-ng          conda-forge/linux-64::libgcc-ng-9.3.0-h2828fa1_18
  libgfortran-ng     conda-forge/linux-64::libgfortran-ng-9.3.0-hff62375_18
  libgfortran5       conda-forge/linux-64::libgfortran5-9.3.0-hff62375_18
  libgomp            conda-forge/linux-64::libgomp-9.3.0-h2828fa1_18
  libiconv           conda-forge/linux-64::libiconv-1.16-h516909a_0
  liblapack          conda-forge/linux-64::liblapack-3.9.0-7_openblas
  libnghttp2         conda-forge/linux-64::libnghttp2-1.41.0-h8cfc5f6_2
  libopenblas        conda-forge/linux-64::libopenblas-0.3.12-pthreads_h4812303_1
  libssh2            conda-forge/linux-64::libssh2-1.9.0-hab1572f_5
  libstdcxx-ng       conda-forge/linux-64::libstdcxx-ng-9.3.0-h6de172a_18
  libta-lib          conda-forge/linux-64::libta-lib-0.4.0-h516909a_0
  ncurses            conda-forge/linux-64::ncurses-6.2-h58526e2_4
  numpy              conda-forge/linux-64::numpy-1.19.5-py38h18fd61f_1
  openssl            conda-forge/linux-64::openssl-1.1.1i-h7f98852_0
  pandas             conda-forge/linux-64::pandas-1.2.0-py38h51da96c_1
  pcre               conda-forge/linux-64::pcre-8.44-he1b5a44_0
  perl               conda-forge/linux-64::perl-5.32.0-h36c2ea0_0
  pip                conda-forge/noarch::pip-20.3.3-pyhd8ed1ab_0
  python             conda-forge/linux-64::python-3.8.6-hffdb5ce_4_cpython
  python-dateutil    conda-forge/noarch::python-dateutil-2.8.1-py_0
  python_abi         conda-forge/linux-64::python_abi-3.8-1_cp38
  pytz               conda-forge/noarch::pytz-2020.5-pyhd8ed1ab_0
  readline           conda-forge/linux-64::readline-8.0-he28a2e2_2
  setuptools         conda-forge/linux-64::setuptools-49.6.0-py38h578d9bd_3
  six                conda-forge/noarch::six-1.15.0-pyh9f0ad1d_0
  sqlite             conda-forge/linux-64::sqlite-3.34.0-h74cdb3f_0
  ta-lib             conda-forge/linux-64::ta-lib-0.4.19-py38hab2c0dc_2
  tk                 conda-forge/linux-64::tk-8.6.10-h21135ba_1
  virtualenv         conda-forge/linux-64::virtualenv-20.4.0-py38h578d9bd_0
  wheel              conda-forge/noarch::wheel-0.36.2-pyhd3deb0d_0
  xz                 conda-forge/linux-64::xz-5.2.5-h516909a_1
  zlib               conda-forge/linux-64::zlib-1.2.11-h516909a_1010


Proceed ([y]/n)? y


Downloading and Extracting Packages
readline-8.0         | 281 KB    | ######################################################### | 100% 
appdirs-1.4.4        | 13 KB     | ######################################################### | 100% 
six-1.15.0           | 14 KB     | ######################################################### | 100% 
liblapack-3.9.0      | 11 KB     | ######################################################### | 100% 
pandas-1.2.0         | 12.0 MB   | ######################################################### | 100% 
krb5-1.17.2          | 1.4 MB    | ######################################################### | 100% 
perl-5.32.0          | 16.8 MB   | ######################################################### | 100% 
ca-certificates-2020 | 137 KB    | ######################################################### | 100% 
certifi-2020.12.5    | 143 KB    | ######################################################### | 100% 
curl-7.71.1          | 139 KB    | ######################################################### | 100% 
distlib-0.3.1        | 246 KB    | ######################################################### | 100% 
setuptools-49.6.0    | 929 KB    | ######################################################### | 100% 
libcurl-7.71.1       | 312 KB    | ######################################################### | 100% 
libiconv-1.16        | 1.4 MB    | ######################################################### | 100% 
gettext-0.19.8.1     | 3.6 MB    | ######################################################### | 100% 
libstdcxx-ng-9.3.0   | 4.0 MB    | ######################################################### | 100% 
zlib-1.2.11          | 106 KB    | ######################################################### | 100% 
tk-8.6.10            | 3.2 MB    | ######################################################### | 100% 
libffi-3.3           | 51 KB     | ######################################################### | 100% 
wheel-0.36.2         | 31 KB     | ######################################################### | 100% 
python_abi-3.8       | 4 KB      | ######################################################### | 100% 
sqlite-3.34.0        | 1.4 MB    | ######################################################### | 100% 
numpy-1.19.5         | 5.4 MB    | ######################################################### | 100% 
xz-5.2.5             | 343 KB    | ######################################################### | 100% 
git-2.30.0           | 12.5 MB   | ######################################################### | 100% 
expat-2.2.9          | 191 KB    | ######################################################### | 100% 
python-3.8.6         | 26.1 MB   | ######################################################### | 100% 
ncurses-6.2          | 985 KB    | ######################################################### | 100% 
c-ares-1.17.1        | 111 KB    | ######################################################### | 100% 
libgfortran-ng-9.3.0 | 22 KB     | ######################################################### | 100% 
libssh2-1.9.0        | 225 KB    | ######################################################### | 100% 
ld_impl_linux-64-2.3 | 617 KB    | ######################################################### | 100% 
openssl-1.1.1i       | 2.1 MB    | ######################################################### | 100% 
python-dateutil-2.8. | 220 KB    | ######################################################### | 100% 
libcblas-3.9.0       | 11 KB     | ######################################################### | 100% 
libgomp-9.3.0        | 376 KB    | ######################################################### | 100% 
ta-lib-0.4.19        | 425 KB    | ######################################################### | 100% 
libblas-3.9.0        | 11 KB     | ######################################################### | 100% 
pcre-8.44            | 261 KB    | ######################################################### | 100% 
_openmp_mutex-4.5    | 22 KB     | ######################################################### | 100% 
libnghttp2-1.41.0    | 774 KB    | ######################################################### | 100% 
libgcc-ng-9.3.0      | 7.8 MB    | ######################################################### | 100% 
libta-lib-0.4.0      | 503 KB    | ######################################################### | 100% 
pytz-2020.5          | 244 KB    | ######################################################### | 100% 
libedit-3.1.20191231 | 121 KB    | ######################################################### | 100% 
pip-20.3.3           | 1.1 MB    | ######################################################### | 100% 
_libgcc_mutex-0.1    | 3 KB      | ######################################################### | 100% 
virtualenv-20.4.0    | 5.0 MB    | ######################################################### | 100% 
libopenblas-0.3.12   | 8.9 MB    | ######################################################### | 100% 
libev-4.33           | 104 KB    | ######################################################### | 100% 
filelock-3.0.12      | 10 KB     | ######################################################### | 100% 
libgfortran5-9.3.0   | 2.0 MB    | ######################################################### | 100% 
Preparing transaction: done
Verifying transaction: done
Executing transaction: done
#
# To activate this environment, use
#
#     $ conda activate freqtrader-env
#
# To deactivate an active environment, use
#
#     $ conda deactivate

$ conda activate freqtrader-env
(freqtrader-env) black@mx:~

```
#2 Download and install freqtrade
```
(base) black@mx:~
$ git clone https://github.com/freqtrade/freqtrade.git^C
(base) black@mx:~
$ conda activate freqtrader-env
(freqtrader-env) black@mx:~
$ git clone https://github.com/freqtrade/freqtrade.git
Cloning into 'freqtrade'...
remote: Enumerating objects: 20, done.
remote: Counting objects: 100% (20/20), done.
remote: Compressing objects: 100% (17/17), done.
remote: Total 54506 (delta 6), reused 8 (delta 3), pack-reused 54486
Receiving objects: 100% (54506/54506), 48.41 MiB | 4.02 MiB/s, done.
Resolving deltas: 100% (40439/40439), done.
(freqtrader-env) black@mx:~
$ cd freqtrade/
(freqtrader-env) black@mx:~/freqtrade
$ ./setup.sh --install
/home/black/miniconda3/envs/freqtrader-env/bin/python3.8
using Python 3.8
-------------------------
Installing mandatory dependencies
-------------------------
Debian/Ubuntu detected. Setup for this system in-progress
Hit:1 http://deb.debian.org/debian buster-updates InRelease
Hit:2 http://deb.debian.org/debian buster InRelease
Hit:3 http://deb.debian.org/debian-security buster/updates InRelease
Hit:4 http://mirrors.rit.edu/mxlinux/mx-packages/mx/repo buster InRelease
Reading package lists... Done
Reading package lists... Done
Building dependency tree       
Reading state information... Done
build-essential is already the newest version (12.6).
git is already the newest version (1:2.20.1-2+deb10u3).
make is already the newest version (4.2.1-1.2).
wget is already the newest version (1.20.1-1.1).
wget set to manually installed.
The following additional packages will be installed:
  autotools-dev m4
Suggested packages:
  autoconf-archive gnu-standards autoconf-doc gettext libtool-doc automaken gfortran | fortran95-compiler gcj-jdk m4-doc
Recommended packages:
  automake | automaken libltdl-dev
The following NEW packages will be installed:
  autoconf autotools-dev libtool m4 pkg-config
0 upgraded, 5 newly installed, 0 to remove and 0 not upgraded.
Need to get 1,232 kB of archives.
After this operation, 3,908 kB of additional disk space will be used.
Get:1 http://deb.debian.org/debian buster/main amd64 m4 amd64 1.4.18-2 [203 kB]
Get:2 http://deb.debian.org/debian buster/main amd64 autoconf all 2.69-11 [341 kB]
Get:3 http://deb.debian.org/debian buster/main amd64 autotools-dev all 20180224.1 [77.0 kB]
Get:4 http://deb.debian.org/debian buster/main amd64 libtool all 2.4.6-9 [547 kB]
Get:5 http://deb.debian.org/debian buster/main amd64 pkg-config amd64 0.29-6 [63.5 kB]
Fetched 1,232 kB in 1s (2,448 kB/s)
Selecting previously unselected package m4.
(Reading database ... 348139 files and directories currently installed.)
Preparing to unpack .../archives/m4_1.4.18-2_amd64.deb ...
Unpacking m4 (1.4.18-2) ...
Selecting previously unselected package autoconf.
Preparing to unpack .../autoconf_2.69-11_all.deb ...
Unpacking autoconf (2.69-11) ...
Selecting previously unselected package autotools-dev.
Preparing to unpack .../autotools-dev_20180224.1_all.deb ...
Unpacking autotools-dev (20180224.1) ...
Selecting previously unselected package libtool.
Preparing to unpack .../libtool_2.4.6-9_all.deb ...
Unpacking libtool (2.4.6-9) ...
Selecting previously unselected package pkg-config.
Preparing to unpack .../pkg-config_0.29-6_amd64.deb ...
Unpacking pkg-config (0.29-6) ...
Setting up m4 (1.4.18-2) ...
Setting up autotools-dev (20180224.1) ...
Setting up pkg-config (0.29-6) ...
Setting up autoconf (2.69-11) ...
Setting up libtool (2.4.6-9) ...
Processing triggers for install-info (6.5.0.dfsg.1-4+b1) ...
install-info: warning: no info dir entry in `/usr/share/info/gtkdialog.info.gz'
Processing triggers for man-db (2.8.5-2) ...
ta-lib/
ta-lib/config.sub
ta-lib/aclocal.m4
ta-lib/CHANGELOG.TXT
ta-lib/include/
ta-lib/include/ta_abstract.h
ta-lib/include/ta_func.h
ta-lib/include/ta_common.h
ta-lib/include/ta_config.h.in
ta-lib/include/Makefile.am
ta-lib/include/ta_libc.h
ta-lib/include/ta_defs.h
ta-lib/missing
ta-lib/ta-lib.spec.in
ta-lib/config.guess
ta-lib/Makefile.in
ta-lib/ta-lib.dpkg.in
ta-lib/Makefile.am
ta-lib/autogen.sh
ta-lib/install-sh
ta-lib/configure
ta-lib/depcomp
ta-lib/HISTORY.TXT
ta-lib/configure.in
ta-lib/autom4te.cache/
ta-lib/autom4te.cache/output.0
ta-lib/autom4te.cache/requests
ta-lib/autom4te.cache/output.1
ta-lib/autom4te.cache/traces.0
ta-lib/autom4te.cache/traces.1
ta-lib/ltmain.sh
ta-lib/ta-lib-config.in
ta-lib/src/
ta-lib/src/ta_func/
ta-lib/src/ta_func/ta_MACDFIX.c
ta-lib/src/ta_func/ta_CDLPIERCING.c
ta-lib/src/ta_func/ta_DIV.c
ta-lib/src/ta_func/ta_ROCR100.c
ta-lib/src/ta_func/ta_ADXR.c
ta-lib/src/ta_func/ta_MAVP.c
ta-lib/src/ta_func/ta_CDLCLOSINGMARUBOZU.c
ta-lib/src/ta_func/ta_COSH.c
ta-lib/src/ta_func/ta_EXP.c
ta-lib/src/ta_func/ta_MINMAXINDEX.c
ta-lib/src/ta_func/ta_SQRT.c
ta-lib/src/ta_func/ta_FLOOR.c
ta-lib/src/ta_func/ta_CDLCONCEALBABYSWALL.c
ta-lib/src/ta_func/ta_NATR.c
ta-lib/src/ta_func/ta_CDLHARAMICROSS.c
ta-lib/src/ta_func/ta_MINUS_DM.c
ta-lib/src/ta_func/ta_LOG10.c
ta-lib/src/ta_func/ta_LINEARREG_ANGLE.c
ta-lib/src/ta_func/ta_RSI.c
ta-lib/src/ta_func/ta_CDLABANDONEDBABY.c
ta-lib/src/ta_func/ta_SAR.c
ta-lib/src/ta_func/ta_CDLBREAKAWAY.c
ta-lib/src/ta_func/ta_CDLDRAGONFLYDOJI.c
ta-lib/src/ta_func/ta_CDLHIGHWAVE.c
ta-lib/src/ta_func/ta_CDLKICKING.c
ta-lib/src/ta_func/ta_CDLDOJISTAR.c
ta-lib/src/ta_func/ta_VAR.c
ta-lib/src/ta_func/ta_CDLMATCHINGLOW.c
ta-lib/src/ta_func/ta_CDLGAPSIDESIDEWHITE.c
ta-lib/src/ta_func/ta_CDLMARUBOZU.c
ta-lib/src/ta_func/ta_AROONOSC.c
ta-lib/src/ta_func/ta_WCLPRICE.c
ta-lib/src/ta_func/ta_CDLEVENINGDOJISTAR.c
ta-lib/src/ta_func/ta_CDL3INSIDE.c
ta-lib/src/ta_func/ta_OBV.c
ta-lib/src/ta_func/ta_AROON.c
ta-lib/src/ta_func/ta_CDLBELTHOLD.c
ta-lib/src/ta_func/ta_CDLSPINNINGTOP.c
ta-lib/src/ta_func/ta_AD.c
ta-lib/src/ta_func/ta_MAX.c
ta-lib/src/ta_func/ta_CDLENGULFING.c
ta-lib/src/ta_func/ta_MINMAX.c
ta-lib/src/ta_func/ta_CDLINNECK.c
ta-lib/src/ta_func/ta_STDDEV.c
ta-lib/src/ta_func/ta_NVI.c
ta-lib/src/ta_func/ta_CDLHAMMER.c
ta-lib/src/ta_func/ta_ASIN.c
ta-lib/src/ta_func/ta_SUM.c
ta-lib/src/ta_func/ta_STOCH.c
ta-lib/src/ta_func/ta_CDLLONGLEGGEDDOJI.c
ta-lib/src/ta_func/ta_MEDPRICE.c
ta-lib/src/ta_func/ta_CDL3STARSINSOUTH.c
ta-lib/src/ta_func/ta_HT_TRENDMODE.c
ta-lib/src/ta_func/ta_BBANDS.c
ta-lib/src/ta_func/ta_CDLMORNINGSTAR.c
ta-lib/src/ta_func/ta_HT_DCPHASE.c
ta-lib/src/ta_func/ta_CDLLONGLINE.c
ta-lib/src/ta_func/ta_TAN.c
ta-lib/src/ta_func/ta_SMA.c
ta-lib/src/ta_func/ta_DX.c
ta-lib/src/ta_func/ta_MIDPOINT.c
ta-lib/src/ta_func/ta_CDL2CROWS.c
ta-lib/src/ta_func/ta_CORREL.c
ta-lib/src/ta_func/ta_CDL3BLACKCROWS.c
ta-lib/src/ta_func/ta_ADD.c
ta-lib/src/ta_func/Makefile.in
ta-lib/src/ta_func/ta_CDLTHRUSTING.c
ta-lib/src/ta_func/ta_SUB.c
ta-lib/src/ta_func/ta_CDLSTALLEDPATTERN.c
ta-lib/src/ta_func/ta_CDLTRISTAR.c
ta-lib/src/ta_func/ta_MA.c
ta-lib/src/ta_func/ta_HT_SINE.c
ta-lib/src/ta_func/ta_ACOS.c
ta-lib/src/ta_func/ta_CDLSTICKSANDWICH.c
ta-lib/src/ta_func/ta_SINH.c
ta-lib/src/ta_func/ta_utility.h
ta-lib/src/ta_func/ta_CDLSHORTLINE.c
ta-lib/src/ta_func/ta_ATAN.c
ta-lib/src/ta_func/ta_CDLADVANCEBLOCK.c
ta-lib/src/ta_func/ta_CDLKICKINGBYLENGTH.c
ta-lib/src/ta_func/ta_CDLSHOOTINGSTAR.c
ta-lib/src/ta_func/ta_ROCR.c
ta-lib/src/ta_func/ta_WMA.c
ta-lib/src/ta_func/ta_CDLDARKCLOUDCOVER.c
ta-lib/src/ta_func/ta_CDLXSIDEGAP3METHODS.c
ta-lib/src/ta_func/ta_TYPPRICE.c
ta-lib/src/ta_func/ta_CDL3WHITESOLDIERS.c
ta-lib/src/ta_func/Makefile.am
ta-lib/src/ta_func/ta_MACDEXT.c
ta-lib/src/ta_func/ta_ADX.c
ta-lib/src/ta_func/ta_PLUS_DM.c
ta-lib/src/ta_func/ta_CDLUPSIDEGAP2CROWS.c
ta-lib/src/ta_func/ta_LN.c
ta-lib/src/ta_func/ta_DEMA.c
ta-lib/src/ta_func/ta_CDL3OUTSIDE.c
ta-lib/src/ta_func/ta_CDLTASUKIGAP.c
ta-lib/src/ta_func/ta_MAMA.c
ta-lib/src/ta_func/ta_CDLMORNINGDOJISTAR.c
ta-lib/src/ta_func/ta_PLUS_DI.c
ta-lib/src/ta_func/ta_MININDEX.c
ta-lib/src/ta_func/ta_COS.c
ta-lib/src/ta_func/ta_HT_TRENDLINE.c
ta-lib/src/ta_func/ta_MIDPRICE.c
ta-lib/src/ta_func/ta_CEIL.c
ta-lib/src/ta_func/ta_TRIMA.c
ta-lib/src/ta_func/ta_CDLSEPARATINGLINES.c
ta-lib/src/ta_func/ta_ROCP.c
ta-lib/src/ta_func/ta_CDLHOMINGPIGEON.c
ta-lib/src/ta_func/ta_CDLHANGINGMAN.c
ta-lib/src/ta_func/ta_AVGPRICE.c
ta-lib/src/ta_func/ta_APO.c
ta-lib/src/ta_func/ta_CDLRISEFALL3METHODS.c
ta-lib/src/ta_func/ta_TRANGE.c
ta-lib/src/ta_func/ta_TSF.c
ta-lib/src/ta_func/ta_LINEARREG.c
ta-lib/src/ta_func/ta_PVI.c
ta-lib/src/ta_func/ta_CDLHIKKAKEMOD.c
ta-lib/src/ta_func/ta_MFI.c
ta-lib/src/ta_func/ta_CDLHARAMI.c
ta-lib/src/ta_func/ta_MACD.c
ta-lib/src/ta_func/ta_BETA.c
ta-lib/src/ta_func/ta_CDLINVERTEDHAMMER.c
ta-lib/src/ta_func/ta_LINEARREG_SLOPE.c
ta-lib/src/ta_func/ta_STOCHF.c
ta-lib/src/ta_func/ta_MIN.c
ta-lib/src/ta_func/ta_CDLIDENTICAL3CROWS.c
ta-lib/src/ta_func/ta_CDLRICKSHAWMAN.c
ta-lib/src/ta_func/ta_T3.c
ta-lib/src/ta_func/ta_CDLMATHOLD.c
ta-lib/src/ta_func/ta_CDLUNIQUE3RIVER.c
ta-lib/src/ta_func/ta_ADOSC.c
ta-lib/src/ta_func/ta_MAXINDEX.c
ta-lib/src/ta_func/ta_ULTOSC.c
ta-lib/src/ta_func/ta_TRIX.c
ta-lib/src/ta_func/ta_MOM.c
ta-lib/src/ta_func/ta_CDLDOJI.c
ta-lib/src/ta_func/ta_EMA.c
ta-lib/src/ta_func/ta_STOCHRSI.c
ta-lib/src/ta_func/ta_ROC.c
ta-lib/src/ta_func/ta_CDLEVENINGSTAR.c
ta-lib/src/ta_func/ta_CDLCOUNTERATTACK.c
ta-lib/src/ta_func/ta_LINEARREG_INTERCEPT.c
ta-lib/src/ta_func/ta_SAREXT.c
ta-lib/src/ta_func/ta_WILLR.c
ta-lib/src/ta_func/ta_MULT.c
ta-lib/src/ta_func/ta_ATR.c
ta-lib/src/ta_func/ta_BOP.c
ta-lib/src/ta_func/ta_CMO.c
ta-lib/src/ta_func/ta_CDLONNECK.c
ta-lib/src/ta_func/ta_CCI.c
ta-lib/src/ta_func/ta_CDLLADDERBOTTOM.c
ta-lib/src/ta_func/ta_HT_PHASOR.c
ta-lib/src/ta_func/ta_utility.c
ta-lib/src/ta_func/ta_PPO.c
ta-lib/src/ta_func/ta_CDLHIKKAKE.c
ta-lib/src/ta_func/ta_HT_DCPERIOD.c
ta-lib/src/ta_func/ta_CDL3LINESTRIKE.c
ta-lib/src/ta_func/ta_TEMA.c
ta-lib/src/ta_func/ta_SIN.c
ta-lib/src/ta_func/ta_MINUS_DI.c
ta-lib/src/ta_func/ta_KAMA.c
ta-lib/src/ta_func/ta_TANH.c
ta-lib/src/ta_func/ta_CDLTAKURI.c
ta-lib/src/ta_func/ta_CDLGRAVESTONEDOJI.c
ta-lib/src/ta_common/
ta-lib/src/ta_common/ta_pragma.h
ta-lib/src/ta_common/ta_magic_nb.h
ta-lib/src/ta_common/ta_retcode.csv
ta-lib/src/ta_common/Makefile.in
ta-lib/src/ta_common/Makefile.am
ta-lib/src/ta_common/ta_memory.h
ta-lib/src/ta_common/ta_version.c
ta-lib/src/ta_common/ta_global.h
ta-lib/src/ta_common/ta_global.c
ta-lib/src/ta_common/ta_retcode.c
ta-lib/src/Makefile.in
ta-lib/src/ta_abstract/
ta-lib/src/ta_abstract/frames/
ta-lib/src/ta_abstract/frames/ta_frame.c
ta-lib/src/ta_abstract/frames/ta_frame.h
ta-lib/src/ta_abstract/excel_glue.c
ta-lib/src/ta_abstract/ta_frame_priv.h
ta-lib/src/ta_abstract/ta_func_api.c
ta-lib/src/ta_abstract/Makefile.in
ta-lib/src/ta_abstract/ta_def_ui.h
ta-lib/src/ta_abstract/Makefile.am
ta-lib/src/ta_abstract/ta_abstract.c
ta-lib/src/ta_abstract/ta_group_idx.c
ta-lib/src/ta_abstract/tables/
ta-lib/src/ta_abstract/tables/table_u.c
ta-lib/src/ta_abstract/tables/table_x.c
ta-lib/src/ta_abstract/tables/table_r.c
ta-lib/src/ta_abstract/tables/table_f.c
ta-lib/src/ta_abstract/tables/table_j.c
ta-lib/src/ta_abstract/tables/table_e.c
ta-lib/src/ta_abstract/tables/table_t.c
ta-lib/src/ta_abstract/tables/table_n.c
ta-lib/src/ta_abstract/tables/table_i.c
ta-lib/src/ta_abstract/tables/table_c.c
ta-lib/src/ta_abstract/tables/table_l.c
ta-lib/src/ta_abstract/tables/table_k.c
ta-lib/src/ta_abstract/tables/table_g.c
ta-lib/src/ta_abstract/tables/table_d.c
ta-lib/src/ta_abstract/tables/table_h.c
ta-lib/src/ta_abstract/tables/table_o.c
ta-lib/src/ta_abstract/tables/table_b.c
ta-lib/src/ta_abstract/tables/table_q.c
ta-lib/src/ta_abstract/tables/table_v.c
ta-lib/src/ta_abstract/tables/table_m.c
ta-lib/src/ta_abstract/tables/table_s.c
ta-lib/src/ta_abstract/tables/table_y.c
ta-lib/src/ta_abstract/tables/table_p.c
ta-lib/src/ta_abstract/tables/table_z.c
ta-lib/src/ta_abstract/tables/table_a.c
ta-lib/src/ta_abstract/tables/table_w.c
ta-lib/src/ta_abstract/ta_def_ui.c
ta-lib/src/ta_abstract/templates/
ta-lib/src/ta_abstract/templates/ta_x.c.template
ta-lib/src/ta_abstract/templates/ta_java_defs.h.template
ta-lib/src/ta_abstract/templates/excel_glue.c.template
ta-lib/src/ta_abstract/templates/ta_group_idx.c.template
ta-lib/src/ta_abstract/templates/ta_frame.c.template
ta-lib/src/ta_abstract/templates/CoreAnnotated.java.template
ta-lib/src/ta_abstract/templates/ta_func.h.template
ta-lib/src/ta_abstract/templates/ta_frame.h.template
ta-lib/src/ta_abstract/templates/Makefile.am.template
ta-lib/src/ta_abstract/templates/ta_func_api.c.template
ta-lib/src/ta_abstract/templates/ta_func.swg.template
ta-lib/src/ta_abstract/templates/ta_retcode.c.template
ta-lib/src/ta_abstract/ta_java_defs.h
ta-lib/src/Makefile.am
ta-lib/src/tools/
ta-lib/src/tools/ta_regtest/
ta-lib/src/tools/ta_regtest/test_util.c
ta-lib/src/tools/ta_regtest/ta_test_func.h
ta-lib/src/tools/ta_regtest/test_data.c
ta-lib/src/tools/ta_regtest/ta_gDataHigh.c
ta-lib/src/tools/ta_regtest/Makefile.in
ta-lib/src/tools/ta_regtest/test_internals.c
ta-lib/src/tools/ta_regtest/Makefile.am
ta-lib/src/tools/ta_regtest/ta_regtest.c
ta-lib/src/tools/ta_regtest/ta_gDataOpen.c
ta-lib/src/tools/ta_regtest/ta_gDataClose.c
ta-lib/src/tools/ta_regtest/test_abstract.c
ta-lib/src/tools/ta_regtest/ta_test_func/
ta-lib/src/tools/ta_regtest/ta_test_func/test_bbands.c
ta-lib/src/tools/ta_regtest/ta_test_func/test_stddev.c
ta-lib/src/tools/ta_regtest/ta_test_func/test_1in_2out.c
ta-lib/src/tools/ta_regtest/ta_test_func/test_sar.c
ta-lib/src/tools/ta_regtest/ta_test_func/test_1in_1out.c
ta-lib/src/tools/ta_regtest/ta_test_func/test_trange.c
ta-lib/src/tools/ta_regtest/ta_test_func/test_macd.c
ta-lib/src/tools/ta_regtest/ta_test_func/test_po.c
ta-lib/src/tools/ta_regtest/ta_test_func/test_per_hlc.c
ta-lib/src/tools/ta_regtest/ta_test_func/test_mom.c
ta-lib/src/tools/ta_regtest/ta_test_func/test_per_ohlc.c
ta-lib/src/tools/ta_regtest/ta_test_func/test_adx.c
ta-lib/src/tools/ta_regtest/ta_test_func/test_candlestick.c
ta-lib/src/tools/ta_regtest/ta_test_func/test_rsi.c
ta-lib/src/tools/ta_regtest/ta_test_func/test_per_ema.c
ta-lib/src/tools/ta_regtest/ta_test_func/test_minmax.c
ta-lib/src/tools/ta_regtest/ta_test_func/test_per_hlcv.c
ta-lib/src/tools/ta_regtest/ta_test_func/test_per_hl.c
ta-lib/src/tools/ta_regtest/ta_test_func/test_stoch.c
ta-lib/src/tools/ta_regtest/ta_test_func/test_ma.c
ta-lib/src/tools/ta_regtest/ta_error_number.h
ta-lib/src/tools/ta_regtest/ta_test_priv.h
ta-lib/src/tools/ta_regtest/ReadMe.txt
ta-lib/src/tools/ta_regtest/ta_gDataLow.c
ta-lib/src/tools/Makefile.in
ta-lib/src/tools/Makefile.am
ta-lib/src/tools/gen_code/
ta-lib/src/tools/gen_code/java/
ta-lib/src/tools/gen_code/java/PrettyCode.java
ta-lib/src/tools/gen_code/java/Main.java
ta-lib/src/tools/gen_code/gen_code.c
ta-lib/src/tools/gen_code/Makefile.in
ta-lib/src/tools/gen_code/Makefile.am
ta-lib/src/tools/gen_code/mcpp.exe
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /usr/bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking for gcc... gcc
checking for C compiler default output file name... a.out
checking whether the C compiler works... yes
checking whether we are cross compiling... no
checking for suffix of executables... 
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ISO C89... none needed
checking for style of include used by make... GNU
checking dependency style of gcc... gcc3
checking build system type... x86_64-unknown-linux-gnu
checking host system type... x86_64-unknown-linux-gnu
checking for a sed that does not truncate output... /usr/bin/sed
checking for grep that handles long lines and -e... /usr/bin/grep
checking for egrep... /usr/bin/grep -E
checking for ld used by gcc... /usr/bin/ld
checking if the linker (/usr/bin/ld) is GNU ld... yes
checking for /usr/bin/ld option to reload object files... -r
checking for BSD-compatible nm... /usr/bin/nm -B
checking whether ln -s works... yes
checking how to recognise dependent libraries... pass_all
checking how to run the C preprocessor... gcc -E
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking dlfcn.h usability... yes
checking dlfcn.h presence... yes
checking for dlfcn.h... yes
checking for g++... g++
checking whether we are using the GNU C++ compiler... yes
checking whether g++ accepts -g... yes
checking dependency style of g++... gcc3
checking how to run the C++ preprocessor... g++ -E
checking for g77... no
checking for xlf... no
checking for f77... no
checking for frt... no
checking for pgf77... no
checking for cf77... no
checking for fort77... no
checking for fl32... no
checking for af77... no
checking for xlf90... no
checking for f90... no
checking for pgf90... no
checking for pghpf... no
checking for epcf90... no
checking for gfortran... no
checking for g95... no
checking for xlf95... no
checking for f95... no
checking for fort... no
checking for ifort... no
checking for ifc... no
checking for efc... no
checking for pgf95... no
checking for lf95... no
checking for ftn... no
checking whether we are using the GNU Fortran 77 compiler... no
checking whether  accepts -g... no
checking the maximum length of command line arguments... 32768
checking command to parse /usr/bin/nm -B output from gcc object... ok
checking for objdir... .libs
checking for ar... ar
checking for ranlib... ranlib
checking for strip... strip
checking if gcc supports -fno-rtti -fno-exceptions... no
checking for gcc option to produce PIC... -fPIC
checking if gcc PIC flag -fPIC works... yes
checking if gcc static flag -static works... yes
checking if gcc supports -c -o file.o... yes
checking whether the gcc linker (/usr/bin/ld -m elf_x86_64) supports shared libraries... yes
checking whether -lc should be explicitly linked in... no
checking dynamic linker characteristics... GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... yes
checking whether to build static libraries... yes
configure: creating libtool
appending configuration tag "CXX" to libtool
checking for ld used by g++... /usr/bin/ld -m elf_x86_64
checking if the linker (/usr/bin/ld -m elf_x86_64) is GNU ld... yes
checking whether the g++ linker (/usr/bin/ld -m elf_x86_64) supports shared libraries... yes
checking for g++ option to produce PIC... -fPIC
checking if g++ PIC flag -fPIC works... yes
checking if g++ static flag -static works... yes
checking if g++ supports -c -o file.o... yes
checking whether the g++ linker (/usr/bin/ld -m elf_x86_64) supports shared libraries... yes
checking dynamic linker characteristics... GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
appending configuration tag "F77" to libtool
checking for dlopen in -ldl... yes
checking for pthread_create in -lpthread... yes
checking for ANSI C header files... (cached) yes
checking float.h usability... yes
checking float.h presence... yes
checking for float.h... yes
checking for inttypes.h... (cached) yes
checking limits.h usability... yes
checking limits.h presence... yes
checking for limits.h... yes
checking locale.h usability... yes
checking locale.h presence... yes
checking for locale.h... yes
checking stddef.h usability... yes
checking stddef.h presence... yes
checking for stddef.h... yes
checking for stdint.h... (cached) yes
checking for stdlib.h... (cached) yes
checking for string.h... (cached) yes
checking for unistd.h... (cached) yes
checking wchar.h usability... yes
checking wchar.h presence... yes
checking for wchar.h... yes
checking wctype.h usability... yes
checking wctype.h presence... yes
checking for wctype.h... yes
checking for an ANSI C-conforming const... yes
checking for size_t... yes
checking whether struct tm is in sys/time.h or time.h... time.h
checking for working volatile... yes
checking for ptrdiff_t... yes
checking return type of signal handlers... void
checking for working strcoll... yes
checking for strftime... yes
checking for working strtod... yes
checking for vprintf... yes
checking for _doprnt... no
checking for floor... no
checking for isascii... yes
checking for localeconv... yes
checking for mblen... yes
checking for memmove... yes
checking for memset... yes
checking for modf... yes
checking for pow... no
checking for sqrt... no
checking for strcasecmp... yes
checking for strchr... yes
checking for strerror... yes
checking for strncasecmp... yes
checking for strrchr... yes
checking for strstr... yes
checking for strtol... yes
checking for strtoul... yes
configure: creating ./config.status
config.status: creating Makefile
config.status: creating src/Makefile
config.status: creating src/ta_abstract/Makefile
config.status: creating src/ta_common/Makefile
config.status: creating src/ta_func/Makefile
config.status: creating src/tools/Makefile
config.status: creating src/tools/gen_code/Makefile
config.status: creating src/tools/ta_regtest/Makefile
config.status: creating ta-lib-config
config.status: creating ta-lib.spec
config.status: creating ta-lib.dpkg
config.status: creating include/ta_config.h
config.status: executing depfiles commands
Making all in src
make[1]: Entering directory '/home/black/freqtrade/build_helpers/ta-lib/src'
Making all in ta_abstract
make[2]: Entering directory '/home/black/freqtrade/build_helpers/ta-lib/src/ta_abstract'
/bin/bash ../../libtool --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I../../include  -I../ta_common/ -Iframes/   -g -O2 -MT libta_abstract_la-ta_group_idx.lo -MD -MP -MF .deps/libta_abstract_la-ta_group_idx.Tpo -c -o libta_abstract_la-ta_group_idx.lo `test -f 'ta_group_idx.c' || echo './'`ta_group_idx.c
mkdir .libs
 gcc -DHAVE_CONFIG_H -I. -I../../include -I../ta_common/ -Iframes/ -g -O2 -MT libta_abstract_la-ta_group_idx.lo -MD -MP -MF .deps/libta_abstract_la-ta_group_idx.Tpo -c ta_group_idx.c  -fPIC -DPIC -o .libs/libta_abstract_la-ta_group_idx.o
 gcc -DHAVE_CONFIG_H -I. -I../../include -I../ta_common/ -Iframes/ -g -O2 -MT libta_abstract_la-ta_group_idx.lo -MD -MP -MF .deps/libta_abstract_la-ta_group_idx.Tpo -c ta_group_idx.c -o libta_abstract_la-ta_group_idx.o >/dev/null 2>&1
mv -f .deps/libta_abstract_la-ta_group_idx.Tpo .deps/libta_abstract_la-ta_group_idx.Plo
/bin/bash ../../libtool --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I../../include  -I../ta_common/ -Iframes/   -g -O2 -MT libta_abstract_la-ta_def_ui.lo -MD -MP -MF .deps/libta_abstract_la-ta_def_ui.Tpo -c -o libta_abstract_la-ta_def_ui.lo `test -f 'ta_def_ui.c' || echo './'`ta_def_ui.c
```

Reply:
when ta-lib was up to continue installation, i stopped it, to get this log

Reply:
this is exactly my point with using the `./setup.sh` command - it'll install certain parts twice (ta-lib is installed once via conda, and once compiled from source).

In my opinion, replacing `./setup.sh -i` with something similar to `pip install -r requirements-dev.txt && pip install -e .` should solve this, as then  pip "should" detect and use the conda-version of ta-lib, which was installed before.

This would also make the conda-guide "os-independent"(ish) - as the `setup.sh` script does not work on windows (at least not outside of a wsl, which however is technically linux) - and i see the strongest usecase for conda on windows, as it'll simplify installation on windows a lot, while on *nix systems, it's "usually" not a huge problem to use the "regular" installation methods if you know the packages you're going to use.